recipes/ffmpeg/all/conanfile.py

```diff
@@ -1,14 +1,20 @@
-from conan.tools.files import rename
-from conans import ConanFile, AutoToolsBuildEnvironment, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os, to_apple_arch
+from conan.tools.env import VirtualBuildEnv, VirtualRunEnv
+from conan.tools.build import check_min_cppstd, cross_building
+from conan.tools.files import copy, get, rm, rename, chdir, rmdir, apply_conandata_patches, export_conandata_patches, replace_in_file
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps, PkgConfigDeps
```

Comment:
You can easily fix linter issue by running it locally (see https://github.com/conan-io/conan-center-index/blob/master/docs/v2_linter.md#running-the-linter-locally)